### DISCUSSION TITLE
Before it arives some thoughts and needs

### DISCUSSION THREAD

**Discussion by edegraaff** [cariboulabs/cariboulite#23] [General]
Hi, 

I can't wait until the Cariboulite will hit my frontdoor and i want to be prepaired, since chips, and pi's are hard to get. 

If i understood well, when i mount the HAT and install the Cariboulabs software i have directly a web-sdr up and running?  That would be great since then you can directly start with receiving all those great things that float trouht space but... 

What if you are familiar with OpenWebRX a great SDR or others?  this brings me to the RPi basic configuration. 

1. OpenWebRX is distributed on a ready to run SD image, wil this work ? 
2. Raspbian image is second good option with the installation  git clone https://github.com/cariboulabs/cariboulite
3. What if you want to make use of Linux based SDR software? like https://gqrx.dk/
4. Will it make use of rtl_tcp and rtl_sdr ? 
4. And what PI do i use, decoding datastreams could be cpu intensive, do i P4? 2, 4, 8 gb mem? 

Questions that are puzzeling me, when this device would be used as replacement for my current RTL-SDR dongles..

I want to be prepaired upfront to make optimal use of the CaribouHAT  all info is welcome

I hope to get some answers here, and who knows it is the start for a lot of amateurs to make use of the CaribouLite in the future.

Regards, 

VERON listening station-nr NL13974
Eelco de Graaff, Voorschoten, Netherlands
https://www.fishfreak.org/sdr
Fly-fish & -tie, Nordics, camp & outdoor

**Comment by anxotato** [comment]
Hi Eelco!
I am also looking forward to receive my Caribou board. Hopefully mid march should arrive (estimated date through mail contact with developers).
I also have similar questions, even a more basic one.
What is the Operative System recommended to install in the Raspberry to use it with Caribou? I would also like to be able to work with other SDRs like Lime SDR an RTL-SDR in the same OS, would that be possible?
Have you make any progress regarding your last year question?

**Reply by edegraaff** [reply]
i did made some progress, if you look arround you will see others have made progress and for me the same, i love to use raspbian, we concludes the best starting point is de fuul.version 64 bit. The installer works ok now in the stable banch, make sure you install the prereq, from there manually install sdr++ and you should getcit working at least at the basics.
